### DISCUSSION TITLE
Failure to setup github copilot 'Node.js not found in PATH'

### DISCUSSION THREAD

**Discussion by harryhanYuhao** [community/community#40300] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have resolved this problem: I have not installed node.js on my computer. (Careless me haven't paid attention to the documentation)  

I am using neovim on ubuntu.
I have installed github copilot as indicated in the official getting-started document using the command:
`git clone https://github.com/github/copilot.vim.git \
  ~/.config/nvim/pack/github/start/copilot.vim`

However, when I type `:Copilot setup` in vim, an error occurred saying: `Node.js not found in PATH`

I would like to ask what is the problem and how to fix it?
Thank you for your help.

**Comment by ossanna16** [comment]
Hi there @harryhanYuhao and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Reply by harryhanYuhao** [reply]
Thank you for your welcome message!

**Comment by harryhanYuhao** [comment, marked as answer]
I just realized the problem: I have to install node.js for it to work.

**Reply by jhaberstroh-sharethis** [reply]
Which you can do with `brew install node.js` :)

**Reply by laurensdc** [reply]
Having a default node configured with `asdf` did _not_ do the trick. 
I had to additionally have a global "default" Node.js installation on my system.
